installer/package.json

```diff
@@ -69,5 +69,13 @@
     "eslint-plugin-jsx-a11y": "^6.5.1",
     "eslint-plugin-react": "^7.27.1",
     "eslint-plugin-react-hooks": "^4.3.0"
+  },
+  "resolutions": {
```

Comment:
This can inject nasty bug, since we will install this version instead of respecting packages dependencies...

Reply:
This is why we have tests, no? 😄